packages/fints/src/__tests__/test-mt940-86-structured.ts

```diff
@@ -47,7 +47,7 @@ const testData = [
             iban: "DE32250400660167600600",
             reference: {
                 customerRef: "1053050069-20180829182432",
-                date: new Date("2018-08-29T18:25:00.000Z"),
+                date: new Date("2018-08-29T18:25:00.000"),
```

Comment:
Please add the `Z`. Without it, this time is local and the tests will behave depending on the timezone of the machine.